### PR TITLE
Added deep links for deprecated entries

### LINKED ISSUE #2079: Deep link deprecated features
As noted in https://github.com/w3c/epub-specs/pull/2060#pullrequestreview-907969509, it'd be useful to deep link deprecated features to where they were last defined. Right now we only refer to the specification.

### PR DESCRIPTION
**Pull request by iherman**
I hope to have found all of them...

Fix #2079

See also [full preview](https://raw.githack.com/w3c/epub-specs/editorial/deep-links-issue-2079/epub33/core/index.html) (including the vocabularies).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/2085.html" title="Last updated on Mar 15, 2022, 3:23 PM UTC (d8dc2c1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/2085/01db0f2...d8dc2c1.html" title="Last updated on Mar 15, 2022, 3:23 PM UTC (d8dc2c1)">Diff</a>

### CODE REVIEW

File: epub33/core/index.html [hunk 1]
```diff
@@ -3846,8 +3846,7 @@ <h5>The <code>meta</code> Element</h5>
 								href="#sec-meta-elem"><code>meta</code> element</a>, which uses different attributes and
 							requires text content, replaces this element.</p>
 
-						<p>For more information about the <code>meta</code> element, refer to its definition in
-							[[OPF-201]].</p>
+						<p>For more information about the <code>meta</code> element, refer to its <a href="http://www.idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section2.2">definition</a> in [[OPF-201]].</p>
```
Comment (wareid) [inline epub33/core/index.html]: For this one, since meta is still a very active part of EPUB, should we not pull the definition into this doc?

**Review by wareid — APPROVED** [review]

File: epub33/core/index.html (hunk 1 above)
Comment (iherman) [reply epub33/core/index.html]: I leave this to @mattgarrish: I "just" added the deep links without any change on the spec.

Comment (mattgarrish) [reply epub33/core/index.html]: This is the OPF2 meta element (the one with name and content attributes). As far as I recall, we've always linked back to EPUB 2 for the legacy elements, so this seems fine to me.

**Review by mattgarrish — COMMENTED** [review]
My one nitpick is going to be accessibility related. The context of the links aren't always apparent ("for more info about this element/property, see its definition..."). I'll push a quick fix for these to put the names in the links.

**Pull request merged**